dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/utils/DataRequestMasker.kt

```diff
@@ -0,0 +1,49 @@
+package org.dataland.datalandcommunitymanager.utils
+import org.dataland.datalandcommunitymanager.model.dataRequest.ExtendedStoredDataRequest
+import org.dataland.datalandcommunitymanager.model.dataRequest.StoredDataRequest
+import org.dataland.keycloakAdapter.auth.DatalandAuthentication
+import org.dataland.keycloakAdapter.auth.DatalandRealmRole
+import org.springframework.stereotype.Service
+
+/**
+ * Implementation of a request admin comment masker
+ *
+ * Masks the admin comment for non admins
+ * */
+@Service
```

Comment:
There is no reason that this class is service.

Reply:
Are you fine with it being a component? Or should I remove @service from all utils classes?

Reply:
The difference between a service and a component is marginal. Both are still managed by the spring context. However, this class does not depend on any other component in the spring service. There is also no managed state by the obect. Therefore, it is sufficient to use a kolin object: https://kotlinlang.org/docs/object-declarations.html